docs/example-scenario/data/big-data-with-iot.md

```diff
@@ -0,0 +1,90 @@
+* [Apache Spark on HDInsight](/azure/hdinsight/spark) supports in-memory processing to boost the performance of big-data analytic applications. In this scenario, Spark is used to run complex algorithms over the data stored in Apache Cassandra.
+
+### Alternatives
+
+* [Cosmos DB](/azure/cosmos-db) is an alternative NoSQL database technology. Cosmos DB provides [multi-master support at global scale](/azure/cosmos-db/multi-region-writers) with [multiple well-defined consistency levels](/azure/cosmos-db/consistency-levels) to meet various customer requirements. It also supports the [Cassandra API](/azure/cosmos-db/cassandra-introduction). 
+* [Data Lake Storage](/azure/storage/data-lake-storage) is an alternative to Blob storage. For this scenario, Data Lake Storage was not available in the targeted region.
+* [Web Apps](/azure/app-service) could also be used to host the web services for ingesting construction results data.
+* Many technology options are available for real-time message ingestion, data storage, stream processing, storage of analytical data, and analytics and reporting. For an overview of these options, their capabilities, and key selection criteria, see [Big data architectures: Real-time processing](/azure/architecture/data-guide/technology-choices/real-time-ingestion) in the [Azure Data Architecture Guide](/azure/architecture/data-guide/).
+   
+## Considerations
+
+The broad availability of Azure regions is an important factor for this scenario. Having more than one region in a single country can provide disaster recovery while also enabling compliance with contractual obligations and law enforcement requirements. Azure's high speed communication between regions is also an important factor in this scenario.
+
+Azure support for open source technologies allowed the customer to take advantage of their existing workforce skills. The customer can also accelerate the adoption of new technologies with significantly lower costs and operating workloads compared to an on-premises solution. 
+
+## Pricing
+
+The following considerations will drive a substantial portion of the costs for this solution.
+
+* Azure virtual machine costs will increase linearly as additional instances are provisioned. Virtual machines that are deallocated will only incur storage costs, and not compute costs. These deallocated machines can then be reallocated when demand is high.
+* [IoT Hub](https://azure.microsoft.com/pricing/details/iot-hub/) costs are driven by the number of IoT units provisioned as well as the service tier chosen, which determines the number of messages per day per unit allowed. 
+* [Stream Analytics](https://azure.microsoft.com/pricing/details/stream-analytics/) is priced by the number of streaming units required to process the data into the service.
+
+## Related Resources
+
+This example scenario is based on a version of this architecture used by [Komatsu][customer-site] For more information, see their [customer story][customer-story]. 
```

Comment:
Let's change this wording to: "To see how an architecture similar to this was implemented, read the Komatsu customer case study."

Reply:
I've updated this. We should stick with the "customer story" terminology/branding though, rather than "case study".